Upstream: yes

From 8596689f2ccda08b06a47263c24dcd97ea0dda14 Mon Sep 17 00:00:00 2001
From: Jonathan Riddell <jr@jriddell.org>
Date: Thu, 21 Aug 2014 12:28:10 +0200
Subject: [PATCH] add version cmake file for PlasmaQuick

add version cmake file for PlasmaQuick
this allows Plasma Desktop to be able to set a required KF5 version

REVIEW:119881
---
 src/plasmaquick/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/plasmaquick/CMakeLists.txt b/src/plasmaquick/CMakeLists.txt
index cc5554a..1ed7c67 100644
--- a/src/plasmaquick/CMakeLists.txt
+++ b/src/plasmaquick/CMakeLists.txt
@@ -111,8 +111,13 @@ ecm_configure_package_config_file(
     PATH_VARS  KF5_INCLUDE_INSTALL_DIR CMAKE_INSTALL_PREFIX
 )
 
+ecm_setup_version(${KF5_VERSION}
+                  VARIABLE_PREFIX PLASMAQUICK
+                  PACKAGE_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/KF5PlasmaQuickConfigVersion.cmake" )
+
 install(FILES
     "${CMAKE_CURRENT_BINARY_DIR}/KF5PlasmaQuickConfig.cmake"
+    "${CMAKE_CURRENT_BINARY_DIR}/KF5PlasmaQuickConfigVersion.cmake"
     DESTINATION "${CMAKECONFIG_INSTALL_DIR}" COMPONENT Devel
 )
 
-- 
2.1.0

